demo/src/index.js

```diff
@@ -75,14 +76,15 @@ function Demo() {
   };
 
   const demo = <ContractEditor
-        lockText={true}
+        lockText={lockTextState}
         value={contractValue}
         onChange={onContractChange}
         editorProps={editorProps}
       />;
 
   return (
     <div>
+      <Button onClick={() => setlockTextState(!lockTextState)} >Click Here</Button>
```

Comment:
I know this is mostly just to help us develop, but it might be more useful for the button text to be "Toggle Lock Text" or something else a little more descriptive. And maybe also render the the current value of lockText so that we can know if it is currently true or false.